### PR TITLE
TCVP-1233 - add Staff API 

### PR DESCRIPTION
**Pull request by pbolduc**
# Description

This PR includes the following proposed change(s):

- Adds TrafficCourts.Staff.Service (Staff API)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/TrafficCourts/Staff.Service/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+WORKDIR "/src/TrafficCourts.Staff.Service"
+RUN dotnet build "TrafficCourts.Staff.Service.csproj" -c Release -o /app/build
+
+FROM build AS publish
+RUN dotnet publish "TrafficCourts.Staff.Service.csproj" -c Release -o /app/publish
+
+FROM base AS final
+WORKDIR /app
+COPY --from=publish /app/publish .
+ENTRYPOINT ["dotnet", "TrafficCourts.Staff.Service.dll"]
```
Comment (KDNeufeld) [inline src/backend/TrafficCourts/Staff.Service/Dockerfile:22]: Add missing EOF newline per github recommendations
```suggestion
ENTRYPOINT ["dotnet", "TrafficCourts.Staff.Service.dll"]

```

File: src/backend/TrafficCourts/Staff.Service/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    },
+    "Docker": {
+      "commandName": "Docker",
+      "launchBrowser": true,
+      "launchUrl": "{Scheme}://{ServiceHost}:{ServicePort}/swagger",
+      "publishAllPorts": true,
+      "useSSL": true
+    }
+  }
+}
```
Comment (KDNeufeld) [inline src/backend/TrafficCourts/Staff.Service/Properties/launchSettings.json:38]:
```suggestion
}

```

**Review by KDNeufeld — APPROVED** [review]
Great, thanks.

File: src/backend/TrafficCourts/Staff.Service/Dockerfile (hunk 1 above)
Comment (pbolduc) [reply src/backend/TrafficCourts/Staff.Service/Dockerfile:22]: That is the default docker file created via the project template

**Pull request merged**